### PR TITLE
Remove archive job icons

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
Whoops, I just saw a few more instances of `hard-drive.png`.  I'll fix these and re-upload.

**Comment by shinnok** [comment]
Hmm, I'm not entirely confident about changing the backup and restore action icons.

The pancake icon for backup is a good choice in my opinion since every-time said action is completed it results in a Tarsnap archive. And for restore, the undo back circle also seems like a good choice to me, rather then simply download.

I am in favor of reducing the many appearances of the pancake icon throughout the UI though. For Jobs, we still need to find a replacement for the hdd icon, even if only for the Job details widget. I'll sit on these changes and see how they grow on me.

File: forms/jobwidget.ui [hunk 1]
```diff
@@ -126,7 +113,7 @@ padding-left: 3px;
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
-          <normaloff>:/icons/tarsnap-icon.png</normaloff>:/icons/tarsnap-icon.png</iconset>
+          <normaloff>:/icons/data-transfer-download.png</normaloff>:/icons/data-transfer-download.png</iconset>
```
Comment (shinnok) [inline forms/jobwidget.ui]: Needs to be upload icon.

**Comment by gperciva** [comment]
I've split off the icon rearrangement into PR #122; I think there's more justification for that part of this PR.

I admit that amount of arrows in the UI is not ideal, so I'll think about this some more and maybe play with some other ideas.  I'm also not convinced that we need any icons in the job or archive details, but I'm not yet confident enough to put forward a PR dealing with those.

**Pull request closed without merging**